### PR TITLE
Prevent a possible bug again that cardinal list going null and affect…

### PR DESCRIPTION
**Pull request by eswordthecat**
… all dependent on it.

### CODE REVIEW

**Comment by PJB3005** [comment]
Jesus christ good thing you noticed this, this could've been a disastrous bug.

**Comment by eswordthecat** [comment]
It's really a disastrous bug before.

**Comment by unid15** [comment]
Can there be constant lists? Might be worth it making cardinal a const

**Comment by eswordthecat** [comment]
If it can open a new PR.

File: code/game/gamemodes/blob/theblob.dm [hunk 1]
```diff
@@ -136,16 +135,15 @@
 	if(istype(T, /turf/space) && prob(75))
 		return
 	if(!T)
-		var/list/dirs = cardinal
```
Comment (N3X15) [inline code/game/gamemodes/blob/theblob.dm]: Oh god I did this didn't I

Fuck

**Comment by N3X15** [comment]
@eswordthecat Just make another commit and push it to the same branch.  It'll update the PR automatically.

**Comment by PJB3005** [comment]
@unid15 you can't make constant lists, since lists are objects, and objects are not seen as constant.

**Comment by eswordthecat** [comment]
@PJB3005 is right, now I remember it doing a constant before but no success.

File: code/game/gamemodes/blob/theblob.dm (hunk 1 above)
Comment (eswordthecat) [inline code/game/gamemodes/blob/theblob.dm]: Spooky references

**Pull request merged**